user-defined-variables.md

```diff
@@ -179,4 +179,9 @@ SELECT @col FROM t;
 +------+
 ```
 
+## MySQL Compatibility
+
+The `SELECT ... INTO <variable>` syntax is supported in MySQL but not in TiDB.
+Other than this the syntax supported by MySQL and TiDB is identical. 
```

Comment:
The linter is complaining about this because it is a manual line break. Please either remove the break or use bullet points.